### PR TITLE
Dev advocate page changes

### PR DESCRIPTION
**Pull request by kimberleyamackenzie**
Implement some updates to the developer advocate team pages + individual pages, [as outlined in the ticket](https://github.com/pulumi/pulumi-hugo/issues/1306).

Most functionality and changes should be visible in the preview link, but a couple things won't be:
- The giveaway card, since no DAs have content in the markdown files for that yet.  See screenshots below with stand-in content
- The "Download" link for slides, since all slides currently are external sites.  See screenshot.

@anita-trimbur After we chatted, I went with the menu-style for the dropdown, but tried to style it as close to the designs as possible.  Let me know what you think.  Also, not included here are a couple of changes to the contact-us hubspot form - I was able to get the UX of the submitted state done, but I need to get access to the form before I can update content.  I'll tackle that on Tuesday.

Ticket: https://github.com/pulumi/pulumi-hugo/issues/1306
Accompanying theme changes here: https://github.com/pulumi/theme/pull/102/
Designs: https://www.figma.com/file/5ob0PX54tpzhVSjBWAmCyV/Developer-Advocate-Pages?node-id=117%3A1422

Screen Shot 2022-04-08 at 4 10 24 PM
Screen Shot 2022-04-08 at 4 10 45 PM
Screen Shot 2022-04-08 at 4 51 01 PM

Screen Shot 2022-04-08 at 4 26 18 PM
Screen Shot 2022-04-08 at 4 27 12 PM

Screen Shot 2022-04-08 at 5 13 11 PM
Screen Shot 2022-04-08 at 5 13 26 PM
Screen Shot 2022-04-08 at 5 13 38 PM
Screen Shot 2022-04-08 at 5 13 46 PM
Screen Shot 2022-04-08 at 5 13 51 PM

### CODE REVIEW

File: themes/default/layouts/community/developer-advocates/single.html [hunk 1]
```diff
@@ -3,63 +3,214 @@
...
+            <div  class="mb-2">Selected Profile</div>
+            <div class="developer-advocate-nav mb-4">
+                <li class="header-nav-item-drop-down">
+                    <span class="dropdown-title whitespace-nowrap">{{ $personData.name }}</span>
+                    <span class="dropdown-menu-caret"></span>
+                    <div class="header-nav-drop-down-menu">
+                        <ul>
+                            {{ range $fullName := $devAdvocates }}
+                                <li>
+                                    {{ $devAdvocatePage := (printf "/community/developer-advocates/%s" ($fullName | urlize)) }}
```
Comment (cnunciato) [inline themes/default/layouts/community/developer-advocates/single.html:21]: This works now, but could easily break or fall out of sync with `team` data. I'd suggest using the same method we use to build the index page (and both would probably be safer with `relref`s, too): https://github.com/pulumi/pulumi-hugo/blob/e3c2318464d2ce98f1af1d0a5a5e6bc95c084da1/themes/default/layouts/community/developer-advocates/list.html#L19-L21

**Review by cnunciato — APPROVED** [review]
One suggestion to make the links a bit safer, otherwise LGTM -- thanks!

**Comment by susanev** [comment]
If it's super easy, I think we should default to being on the past tab when the upcoming has zero items. If it's not super easy let me know and I'll open a separate issue.

**Review by zchase — APPROVED** [review]
LGTM outside of @cnunciato comment.

**Comment by mattstratton** [comment]
I actually find the upcoming/past behavior really confusing. 

image

To me, "Upcoming" in this view looks like a clickable element (maybe because I'm so old that blue = hyperlinkto me? lol)

so I was not clear which actual tab I was on; my brain wants that to be me on the "past" tab, but it's not.

File: themes/default/content/community/developer-advocates/matt-stratton.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Matt Stratton
```
Comment (mattstratton) [inline themes/default/content/community/developer-advocates/matt-stratton.md:2]:
```suggestion
title: Matty Stratton

```
Yes, I know, I'm the one who made the original file with the name this way...lol

File: themes/default/content/community/developer-advocates/matt-stratton.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Matt Stratton
 id: matt-stratton
+first_name: Matt
```
Comment (mattstratton) [inline themes/default/content/community/developer-advocates/matt-stratton.md:4]:
```suggestion
first_name: Matty

```

File: themes/default/layouts/community/developer-advocates/single.html (hunk 1 above)
Comment (mattstratton) [reply themes/default/layouts/community/developer-advocates/single.html:21]: also I could be wrong but with me wanting my displayed name to be "Matty Stratton" but the file is `matt-stratton` that also maybe is weird?

**Comment by susanev** [comment]
> I actually find the upcoming/past behavior really confusing.
> 
> image
> 
> To me, "Upcoming" in this view looks like a clickable element (maybe because I'm so old that blue = hyperlinkto me? lol)
> 
> so I was not clear which actual tab I was on; my brain wants that to be me on the "past" tab, but it's not.

this is a really good point, we should be able to adjust the visual presentation here to address this. will leave it to Anita though.

**Comment by mattstratton** [comment]
FYI I also have a PR in that updates some of my content, which might conflict here, so we should merge or something together :)

https://github.com/pulumi/pulumi-hugo/pull/1350

**Comment by susanev** [comment]
@mattstratton if ur additions are time-sensitive, feel free to merge as we wont be picking this back up until later today.

**Comment by mattstratton** [comment]
> if ur additions are time-sensitive, feel free to merge as we wont be picking this back up until later today.

yah, I wasn't quite clear in my point which was more to say that I was gonna be merging that pr probably today for my talk tomorrow, so we just need to make sure to rebase this one

**Comment by anita-trimbur** [comment]
@mattstratton Totally hear your point. Part of this is that we need to gray out the text-color for the unselected tab which will help with the visual distinction. What we can also do is move away from the brand-blue color accent on the selected tab and use an underline (similar to what we do in Docs)

Group 383

**Comment by nimbinatus** [comment]
Small request: Can we change the name of the `unisex-shirt.jpg` to `loose-shirt.jpg`, `classic-shirt.jpg`, or something similar?

Also, are the hubspot form things all contained in hubspot, so I have to go into there to edit the text? I haven't seen this text yet.

**Comment by anita-trimbur** [comment]
Also is this something that just happens on my computer? Or something to do with the previews? Sometimes the icons disappear on me
Screen Shot 2022-04-12 at 3 49 31 PM

File: themes/default/content/community/developer-advocates/matt-stratton.md [hunk 4]
```diff
@@ -1,6 +1,9 @@
 ---
-title: Matt Stratton
+title: Matty Stratton
+
```
Comment (kimberleyamackenzie) [inline themes/default/content/community/developer-advocates/matt-stratton.md:3]:
```suggestion

```

File: themes/default/content/community/developer-advocates/matt-stratton.md [hunk 5]
```diff
@@ -1,6 +1,9 @@
 ---
-title: Matt Stratton
+title: Matty Stratton
+
 id: matt-stratton
+first_name: Matty
+
```
Comment (kimberleyamackenzie) [inline themes/default/content/community/developer-advocates/matt-stratton.md:6]:
```suggestion

```

File: themes/default/content/community/developer-advocates/matt-stratton.md [hunk 6]
```diff
@@ -38,8 +39,11 @@ talks:
     slides: https://speaking.mattstratton.com/Wwwhs3/kubernetes-you-rainbow-infused-space-unicorn-what-leslie-knope-can-teach-us-about-devops
 form:
     hubspot_form_id: 87c6f1d4-a928-4298-b4b4-89164561e01b
+giveaway:
+    hubspot_form_id: d1360304-bd20-4d03-a635-90caf1e9e1cb
 ---
-Matt Stratton is a Staff Developer Advocate at Pulumi, founder and co-host of the popular [Arrested DevOps](https://www.arresteddevops.com/) podcast, and the global chair of the [DevOpsDays](https://www.devopsdays.org/) set of conferences.
+Matty Stratton is a Staff Developer Advocate at Pulumi, founder and co-host of the popular [Arrested DevOps](https://www.arresteddevops.com/) podcast, and the global chair of the [DevOpsDays](https://www.devopsdays.org/) set of conferences.
+
```
Comment (kimberleyamackenzie) [inline themes/default/content/community/developer-advocates/matt-stratton.md:46]:
```suggestion

```

**Pull request merged**